manifest/v1alpha/alertpolicy/measurement.go

```diff
@@ -63,6 +63,16 @@ func GetExpectedOperatorForMeasurement(measurement Measurement) (v1alpha.Operato
 	}
 }
 
+// getExpectedOperatorForMeasurement returns the operator that should be paired with a given measurement.
+func getExpectedOperatorForMeasurement(measurement Measurement) (v1alpha.Operator, error) {
```

Comment:
I don't think that anyone except Nobl9 use this method to get default operator which is optional, but we could leave it as public. I think the risk that anyone use it is very low and we can try to change it to private and wait for possible feedback.